source/includes/administration/_resource_commitments.md

```diff
@@ -0,0 +1,343 @@
+      "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e"
+    },
+    "version": 1,
+    "deleted": false
+  }, 
+  {
+    "id": "51c01cfe-f0d5-4a3d-841d-35ec2b7df9cc",
+    "displayName": "CRB-SYSTEM-2",
+    ....
+  }]
+}
+```
+List all the resource commitments that you have access to. You can only view resource commitments if you have been assingned either the `View usage` or the `Manage commitments` permission. If you have `View usage` access then you will only be able to see the commitments specific to your organization. Having the `Manage commitments` permission lets you see all resource commitments in the system.
+
+An array of resource commitments with the following attributes are returned.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the resource commitment
+`displayName`<br/>*string* | The name of the resource commitment
+`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the resource commitment was created
+`startDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) from when the resource commitment becomes active
+`endDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) until when the resource commitment will be active<br/>*optional*: `if not present then the commitment has no end date`
+`pricingMethod`<br/>*string* | One of the two following options that define how the effective price for the resource commitment is calculated<br/>*FIXED_MONTHLY*: `a fixed price per billing cycle`<br/>*VARIABLE_UTILITY_DISCOUNT*: `pricing with custom discount on specific resources`
+`currency`<br/>*string* | The currency type based on which the pricing is defined
+`monthlyPrice`<br/>*number* | The effective monthly price for this resource commitment
+`effectiveDiscount`<br/>*number* | The effective discount on this resource commitment
+`billableHoursPerMonth`<br/>*number* | The number hours per month based on which the monthly price is calculated<br/>*optional*: `not present if the pricing method is FIXED_MONTHLY`
+`billingDay`<br/>*number* | The day of the month based on which the billing cycle is calculated
+`resourceCommitmentDetails`<br/>*Array [Commitment detail]* | The resources on which different commitment levels _(and discounts)_ have been set.<br/>*includes*: `id`, `name`, `primaryType`, `secondaryType`, `commitment`, `discountPercent`, `unit`, `labelKey`
+`organization`<br/>*[Organization](#administration-organizations)* | The organization on whose connection the resource commitment is defined<br/>*includes*: `id`, `name`
+`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection on which the resource commitment is defined<br/>*includes*: `id`, `name`, `type`
+
+
+<!-------------------- GET A RESOURCE COMMITMENT -------------------->
+
+### Retrieve a resource commitment
+
+`GET /resource_commitments/:id`
+
+```shell
+# Retrieve visible environment
```

Comment:
Wrong comment here too, you can check others they are all not ok.

Reply:
Fixed in 738496526a7aa1f45bc86f95f7c32cc0e4ed3320